x-pack/test/security_solution_cypress/cypress/e2e/data_sources/sourcerer.cy.ts

```diff
@@ -121,7 +122,7 @@ describe('Sourcerer', () => {
       cy.get(SOURCERER.saveButton).should('be.disabled');
     });
 
-    it(
+    it.skip(
```

Comment:
Same here, is it broken in ESS? Just wondering why we disable it for ess as well

Reply:
unskipped, checking